editor/src/components/navigator/left-pane.tsx

```diff
@@ -545,12 +746,21 @@ const ProjectPane = betterReactMemo('ProjectSettingsPanel', () => {
       userState: store.userState,
       focusedPanel: store.editor.focusedPanel,
       minimised: store.editor.projectSettings.minimised,
+      forkedFrom: store.editor.forkedFromProjectId,
     }
   }, 'ProjectSettingsPanel')
 
   const [name, changeProjectName] = React.useState(projectName)
   const [description, changeProjectDescription] = React.useState(projectDescription)
   const [requestingPreviewImage, setRequestingPreviewImage] = React.useState(false)
+  const { title: forkedFromTitle } = useGetProjectMetadata(forkedFrom)
+
+  const forkedFromText =
```

Comment:
This looks like it's duplicated from earlier.

Reply:
I think it's fine. the copy might diverge, turning this into a component would be premature IMO